invenio_rdm_records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -123,19 +123,152 @@
 
       }
     },
-    "_created": {
+    "dates": {
+      "description": "Date interval.",
+      "items": {
+        "additionalProperties": false,
+        "properties": {
+          "description": {
+            "description": "Description of the date interval.",
+            "type": "string"
+          },
+          "end": {
+            "description": "End date.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "start": {
+            "description": "Start date.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "type": {
+            "description": "Type of the date interval."
+          }
+        },
+        "required": [
+          "type"
+        ],
+        "type": "object"
+      },
+      "type": "array"
+    },
+    "description": {
+      "description": "Description/abstract for record.",
+      "type": "string"
+    },
+    "embargo_date": {
+      "description": "Embargo date of record (ISO8601 formatted date).",
+      "title": "Embargo Date",
+      "type": "string",
+      "format": "date-time"
+    },
+    "keywords": {
+      "description": "Free text keywords.",
+      "items": {
+        "type": "string"
+      },
+      "type": "array"
+    },
+    "language": {
+      "description": "Primary language of the resource. ISO 639-3 language code.",
+      "type": "string"
+    },
+    "owners": {
+      "description": "List of user IDs that are owners of the record.",
+      "items": {
+        "type": "number"
+      },
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "publication_date": {
+      "description": "Record publication date (IS8601-formatted). EDTF support to be added for field.",
+      "type": "string",
+      "format": "date-time"
+    },
+    "recid": {
+      "description": "Invenio record identifier (integer).",
+      "type": "number"
+    },
+    "resource_type": {
+      "additionalProperties": false,
+      "description": "Record resource type.",
+      "properties": {
+        "openaire_subtype": {
+          "description": "OpenAIRE-specific resource type.",
+          "type": "string"
+        },
```

Comment:
normal: I think `openaire` should fit inside a generic resource type which we can deal with in November with the customization work.

Reply:
You are right, removing...